.github/workflows/pylint.yaml

```diff
@@ -0,0 +1,29 @@
+name: Pylint Code analysis
+
+on:
+  push:
+    branches: ["*"]
```

Comment:
nit: I think specifying branch here is not necessary (the default is to run on all branches)

Reply:
Yes! I think this is what I did by setting the `branches:` to `*`

It should run on all branches....

Or you want me to update something ? @baimamboukar

Reply:
I think you can remove `branches: ["*"]` entry altogether - it will work exactly the same

Reply:
alright!... I'm on it

Reply:
It is done ✅🚀